### PR TITLE
Add prac_06 files

### PR DESCRIPTION
**Pull request by Quinn368**
@Ajeetha0701
@ChristineYamin

### CODE REVIEW

**Comment by Ajeetha0701** [comment]
code is good
________________________________
From: Quinn368 ***@***.***>
Sent: 22 March 2024 21:30
To: Quinn368/cp1404practicals ***@***.***>
Cc: Ajeetha Feross ***@***.***>; Mention ***@***.***>
Subject: [Quinn368/cp1404practicals] Add prac_06 files (PR #3)


@Ajeetha0701<https://github.com/Ajeetha0701>
@ChristineYaminoo

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Quinn368/cp1404practicals/pull/3

Commit Summary

  *   89694e7<https://github.com/Quinn368/cp1404practicals/pull/3/commits/89694e7c59d2030c2fc33ad4197d1375d526fc59> Add prac_06 files

File Changes

(7 files<https://github.com/Quinn368/cp1404practicals/pull/3/files>)

  *   A prac_06/car.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-39bbe6854a06ea46c5adcee0f4278757510eef15844514ed9f57f29ea0b4b421> (35)
  *   A prac_06/guitar.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-adfcd4b144c5617537440d8f25e67efa12452ea8b749e39abfc98c6a6a46e832> (21)
  *   A prac_06/guitar_test.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-5082beac2c5f7abaf47626b4dc114dfe762ae88beaba5aafe3ac44b6f7e70c3d> (21)
  *   A prac_06/guitars.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-f413aea8757b4619969b42db51c2f45574baea566a127c31a726f7b64b6930bc> (30)
  *   A prac_06/languages.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-bce9fe3f710a8d2edd79d3b43cc01cb8557b5f3f94b30755ccfa9a93419d79ac> (18)
  *   A prac_06/programming_language.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-afb8f89d9713c145c69e25ad93608dd15fcdd1ef4242161ba0a19c20b4e26fd8> (15)
  *   A prac_06/used_cars.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-1e96d2ed09347ac02fd80b2062599ed397547ec20dfd6d2008d7172dd653491c> (24)

Patch Links:

  *   https://github.com/Quinn368/cp1404practicals/pull/3.patch
  *   https://github.com/Quinn368/cp1404practicals/pull/3.diff

**Comment by Ajeetha0701** [comment]
great job!
________________________________
From: Quinn368 ***@***.***>
Sent: 22 March 2024 21:30
To: Quinn368/cp1404practicals ***@***.***>
Cc: Ajeetha Feross ***@***.***>; Mention ***@***.***>
Subject: [Quinn368/cp1404practicals] Add prac_06 files (PR #3)


@Ajeetha0701<https://github.com/Ajeetha0701>
@ChristineYaminoo

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Quinn368/cp1404practicals/pull/3

Commit Summary

  *   89694e7<https://github.com/Quinn368/cp1404practicals/pull/3/commits/89694e7c59d2030c2fc33ad4197d1375d526fc59> Add prac_06 files

File Changes

(7 files<https://github.com/Quinn368/cp1404practicals/pull/3/files>)

  *   A prac_06/car.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-39bbe6854a06ea46c5adcee0f4278757510eef15844514ed9f57f29ea0b4b421> (35)
  *   A prac_06/guitar.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-adfcd4b144c5617537440d8f25e67efa12452ea8b749e39abfc98c6a6a46e832> (21)
  *   A prac_06/guitar_test.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-5082beac2c5f7abaf47626b4dc114dfe762ae88beaba5aafe3ac44b6f7e70c3d> (21)
  *   A prac_06/guitars.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-f413aea8757b4619969b42db51c2f45574baea566a127c31a726f7b64b6930bc> (30)
  *   A prac_06/languages.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-bce9fe3f710a8d2edd79d3b43cc01cb8557b5f3f94b30755ccfa9a93419d79ac> (18)
  *   A prac_06/programming_language.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-afb8f89d9713c145c69e25ad93608dd15fcdd1ef4242161ba0a19c20b4e26fd8> (15)
  *   A prac_06/used_cars.py<https://github.com/Quinn368/cp1404practicals/pull/3/files#diff-1e96d2ed09347ac02fd80b2062599ed397547ec20dfd6d2008d7172dd653491c> (24)

Patch Links:

  *   https://github.com/Quinn368/cp1404practicals/pull/3.patch
  *   https://github.com/Quinn368/cp1404practicals/pull/3.diff

File: prac_06/languages.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+    languages = [python, ruby, visual_basic]
+
+    print("The dynamically typed languages are:")
+    for language in languages:
+        if language.is_dynamic():
+            print(language.field)
+
+
+main()
```
Comment (ChristineYamin) [inline prac_06/languages.py:18]: Nice coding

File: prac_06/guitars.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        add_guitar = Guitar(name, year, cost)
+        guitars.append(add_guitar)
+        print(f"{add_guitar.name} ({add_guitar.year}) : ${add_guitar.cost:,.2f} added.")
+        name = input("Name: ")
+
+    print("These are my guitars: ")
+    for i, guitar in enumerate(guitars, 1):
```
Comment (ChristineYamin) [inline prac_06/guitars.py:23]: great coding

**Pull request merged**